disnake/enums.py

```diff
@@ -634,6 +638,19 @@ def __int__(self):
         return self.value
 
 
+class TextInputStyle(Enum):
+    short = 1
+    paragraph = 2
+
+    # Aliases
+    singleline = 1
+    long = 2
+    multiline = 2
```

Comment:
I don't think we should have these aliases here.

Reply:
I suppose they might be fine, but I think it should be single_line and multi_line